web/docs/project/env-vars.md

```diff
@@ -104,19 +104,20 @@ Defining environment variables in this way can be cumbersome even for a single p
 
 ## Defining Env Vars in Production
 
-While in development, we had the option of using `.env` files which made it easy to define and manage env vars. However, in production, we need to provide env vars differently.
-
 ![Env vars usage in development and production](/img/env/prod_dev_fade_2.svg)
 
-### Client Env Vars
+While in development, we had the option of using `.env.client` and `.env.server` files which made it easy to define and manage env vars.
+However, for production, `.env.client` and `.env.server` files will be ignored, and we need to provide env vars differently.
 
-Client env vars are embedded into the client code during the build and shipping process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys).
+### Client Env Vars
 
-You should provide them to the build command, for example:
+To set client env vars for production, you need to ensure they are set for the terminal session in which you are running the build command, for example:
```

Comment:
I made it more clear what is happening here, so if you arrive here by hyperlink and read just these few lines, it is clear even without much context (I was in this situation and was confused).